app/views/application/search/select-expertise-type.html

```diff
@@ -48,11 +48,6 @@
             ({{ data.selectedLevel }})
           {% endif %}
           {% else %} 
-          {% if data.selectedLevel == "I don't know the level" %}
```

Comment:
here we need to leave the code
`({{ data.selectedQualification }})` 
so that the qual type still shows

Reply:
this is the same on all of the pages

Reply:
There's a few other parts in the `captionHeading` that need updating based on whether the user selects a subject that has the qual type predefined or not. 

It will be the same as what is currently in the select-level.html file, so can just copy the whole captionheading from there and replace.